completions/zsh/_arch-audit

```diff
@@ -0,0 +1,14 @@
+#compdef arch-audit
+
+typeset -A opt_args
+
+local context state line
+
+_arguments -s -S \
+"(-h --help)"{-h,--help}"[Prints help information]" \
+"(-q|--quiet)"{-q,--quiet}"[Show only vulnerable package names and their versions.]" \
```

Comment:
I guess `-q|--quite` is a typo?

Also, what about `-qq` it's possible to have?